pkg/subctl/cmd/unexport.go

```diff
@@ -0,0 +1,85 @@
+/*
+SPDX-License-Identifier: Apache-2.0
+
+Copyright Contributors to the Submariner project.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/submariner-io/submariner-operator/pkg/subctl/cmd/utils"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	mcsclient "sigs.k8s.io/mcs-api/pkg/client/clientset/versioned/typed/apis/v1alpha1"
+)
+
+var (
+	unexportCmd = &cobra.Command{
+		Use:   "unexport",
+		Short: "Stop a resource from being exported to other clusters",
+		Long:  "This command stops exporting a resource so it isn't accessible to other clusters any more",
+	}
+	unexportServiceCmd = &cobra.Command{
+		Use:   "service <serviceName>",
+		Short: "Stop a Service from being exported to other clusters",
+		Long: "This command removes the ServiceExport resource with the given name which in turn stops the Service " +
+			"of the same name from being exported to other clusters",
+		PreRunE: restConfigProducer.CheckVersionMismatch,
+		Run:     unexportService,
+	}
+)
+
+func init() {
+	restConfigProducer.AddKubeContextFlag(unexportCmd)
+	addNamespaceFlag(unexportServiceCmd)
+	unexportCmd.AddCommand(unexportServiceCmd)
+	rootCmd.AddCommand(unexportCmd)
+}
+
+func unexportService(cmd *cobra.Command, args []string) {
+	err := validateUnexportArguments(args)
+	utils.ExitOnError("Insufficient arguments", err)
```

Comment:
In case you respin, I would recommend using `exit.OnErrorWithMessage` because that is replacing `utils.Exit*` functions.

Reply:
Is this merged?

Reply:
OK found it but the location is weird IMHO, nevertheless I'll update the usages in this file.

You should add a deprecation warning on the utils function though if it's being replaced

Reply:
We could add deprecation but the `utils` functions will be removed once the restructuring is complete next release.  Even if you don't use the `exit` functions now, they'll be converted once this code is moved under `cmd`.